test/datasets/graphs.jl

```diff
@@ -278,3 +278,23 @@ end
     @test size(g.edge_data.features) == (2, g.num_edges)
     @test size(g.edge_data.targets) == (g.num_edges,)
 end
+
+@testset "AQSOL" begin
+    split_counts = Dict(:train => 7985, :val => 998, :test => 999)
+    for split in [:train, :val, :test]
+        data = AQSOL(split=split)
+        @test data isa AbstractDataset
+        @test data.split == split
+        @test length(data) == data.metadata["n_observations"]
+        @test length(data.graphs) == split_counts[split]
+
+        for (i, g) in enumerate(data)
```

Comment:
testing every single graph is too slow. We can test a random one with
```julia
g = rand(data)
```
or something similar.